homeassistant/components/energyzero/sensor.py

```diff
@@ -49,6 +49,7 @@ class EnergyZeroSensorEntityDescription(
     EnergyZeroSensorEntityDescription(
         key="current_hour_price",
         name="Current gas market price",
+        has_entity_name=True,
```

Comment:
If this applies to all entities, then set it once in the base class as `_attr_has_entity_name = True` and remove it from all entity descriptions.